lxc-binding.c

```diff
@@ -67,6 +67,14 @@ int go_lxc_seccomp_notify_fd(struct lxc_container *c) {
 #endif
 }
 
+int go_lxc_devpts_fd(struct lxc_container *c) {
+#if VERSION_AT_LEAST(4, 1, 0)
```

Comment:
Should be 4.0.4 I think, unless there's a reason we don't want this backported?

Reply:
Ah, you're right!

Reply:
Fixed.